examples/rust/get_started/examples/06-secure-channel-many-hops.rs

```diff
@@ -8,22 +8,23 @@ async fn main(mut ctx: Context) -> Result<()> {
     // Start an Echoer worker at address "echoer"
     ctx.start_worker("echoer", Echoer).await?;
 
-    let mut bob = Entity::create(&ctx).await?;
+    let mut bob = Entity::create(&ctx)?;
 
     // Start 3 hop workers at addresses "h1", "h2" and "h3".
     ctx.start_worker("h1", Hop).await?;
     ctx.start_worker("h2", Hop).await?;
     ctx.start_worker("h3", Hop).await?;
 
-    // Create a secure channel listener at address "secure_channel_listener"
+    // TODO JDS Secure Channel integration point
```

Comment:
Do we want to fix that before merging to develop?

Reply:
Definitely, I'll go through and fix all `TODO JDS` first